README.md

```diff
@@ -152,7 +165,7 @@ databases:           # database connections
     jdbc_user: hdfs                                                                   # database user
     jdbc_password: na                                                                 # database password
     jdbc_pooling: false                                                               # (optional) should connection pooling be used (it does not work for Hive due to driver issues)
-    jdbc_jar: tempto-hive-jdbc/build/libs/hive-jdbc-fat.jar                   # (optional) jar to be used for obtaining database driver. Should be used in case when we cannot have it in global classpath due to class conflicts. (e.g. hive driver conflicts with presto driver)
+    jdbc_jar: tempto-hive-jdbc/build/libs/hive-jdbc-fat.jar                           # (optional) jar to be used for obtaining database driver. Should be used in case when we cannot have it in global classpath due to class conflicts.
```

Comment:
(optional) Path to jar containing database driver.  Required if jar is not present in global classpath.